sharktank_models/llama3.1/test_llama.py

```diff
@@ -250,5 +315,75 @@ def test_prefill_decode_hip(toy_llama_hip):
     cross_entropy = prefill_decode_cross_entropy(toy_llama_hip, ids)
     cross_entropy = cross_entropy.item()
     assert cross_entropy == pytest.approx(
-        0.589, 1e-3
+        0.589, 1e-2
+    ), "cross entropy outside of tolerance"
+
+
+@pytest.mark.target_cpu
```

Comment:
You could parameterize these tests across sharding levels (and cpu/hip), rather than copy/paste

https://docs.pytest.org/en/stable/how-to/parametrize.html

Reply:
I reworked the pytest fixtures to better grow configurations. It keeps the same target / environment variable initialization but makes it easier to expand if we choose to. Should be useful for adding grok variants as well.